src/main/resources/hudson/maven/MavenModuleSet/configure-entries.properties

```diff
@@ -22,6 +22,7 @@
 
 Maven\ Version.error.1=Jenkins needs to know where your Maven is installed.
 Maven\ Version.error.2=Please do so from <a href="{0}/configure" target="_new">the system configuration</a>.
+Maven\ Version.error.3=Please do so from <a href="{0}/configureTools" target="_new">the system configuration</a>.
```

Comment:
@abayer For Jenkins 2 shouldn't we call this "the tools configuration" instead ?